### PR TITLE
[LHDM-2201] Header/Footer Custom classes/IDs

### PR DESCRIPTION
**Pull request by seesleestak**
## Summary

- Our team has a specific Smartling-related requirement to be able to assign class names and IDs to the logo, language switch link, and log in link in the Header and the logo in the Footer.
- https://jira.cms.gov/browse/LHDM-2201 

**Update:** After chatting, this has been simplified to only include the custom classes.

## How to test

Use the story controls to apply the various new props. (Note that you'll have to use both the logged in and logged out stories to see all the props.)

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.
- [x] Selected appropriate release milestone

### If this is a change to design:

- [ ] If visual regression image references have been changed, design **MUST** be assigned to review. In this instance, **designer approval is a requirement** before the PR can be merged.

### If this is a change to code:

- [x] Created or updated unit tests to cover any new or modified code
- [x] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [x] Checked for spelling and grammatical errors

### CODE REVIEW

**Comment by seesleestak** [comment]
I ran `yarn test:browser:update` but the changes I saw locally didn't seem related to the PR changes or the build error. Any help deciphering that would be appreciated.

File: packages/ds-healthcare-gov/src/components/Header/__snapshots__/defaultMenuLinks.test.js.snap [hunk 1]
```diff
@@ -13,6 +13,7 @@ Object {
     },
     Object {
       "ariaLabel": "CuidadoDeSalud.gov en Español",
+      "className": undefined,
```
Comment (pwolfert) [inline packages/ds-healthcare-gov/src/components/Header/__snapshots__/defaultMenuLinks.test.js.snap:16]: Having these here is kind of noisy, but I don't think there's any way they'll hurt anything. Using code to omit them would make the component code less legible, so this is a fine tradeoff.

**Comment by pwolfert** [comment]
@zarahzachz, can you review this now that I've made changes?

File: packages/ds-healthcare-gov/src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -29,6 +29,8 @@ export interface FooterProps {
    * Element to be added to display content for top footer section changes
    */
   footerTop?: React.ReactNode;
+  logoId?: string;
```
Comment (zarahzachz) [inline packages/ds-healthcare-gov/src/components/Footer/Footer.tsx:32]: is `logoId` being used somewhere?

**Review by zarahzachz — APPROVED** [review]
one comment. couldn't tell if a prop was being used or not. i might've overlooked something?

File: packages/ds-healthcare-gov/src/components/Footer/Footer.tsx (hunk 2 above)
Comment (pwolfert) [reply packages/ds-healthcare-gov/src/components/Footer/Footer.tsx:32]: Good catch. I missed this and should have removed it.

**Pull request merged**